pkg/apis/v1alpha1/vpc_types.go

```diff
@@ -47,6 +48,8 @@ type VPCStatus struct {
 	LBSubnetPath string `json:"lbSubnetPath"`
 	// CIDR for the load balancer Subnet.
 	LBSubnetCIDR string `json:"lbSubnetCIDR"`
+        // Private CIDRs used for the VPC.
+        PrivateIPv4CIDRs []string `json:"privateIPv4CIDRs"`
```

Comment:
I remember we need to add the usage of the CIDR, not only the list

Reply:
as discussed, the usage of the CIDR will be added if needed in future versions.